View_Assist_custom_sentences/Play_Music_with_Music_Assistant/blueprint-playmusicwithmusicassistant.yaml

```diff
@@ -24,156 +24,192 @@ blueprint:
-            response_variable: found_song
-          - service: music_assistant.play_media
-            target:
-              entity_id: "{{ target_musicplayer_device }}"
-            data:
-              media_type: track
-              enqueue: next
-              media_id: "{{ found_song.tracks.0.name }}"
-              artist: "{{ trigger.slots.artist  }}"
-          - set_conversation_response: ""
+            response_variable: found_track
+            action: music_assistant.search
+          - alias: Did we find a track?
+            if:
+              - condition: template
+                value_template: "{{found_track['tracks'] | count>0}}"
+                alias: We found a track!
+            then:
+              - data:
+                  entity_id: "{{ target_satellite_device }}"
+                  mode: music
+                  title: ""
+                action: python_script.set_state
+              - metadata: {}
+                data:
+                  path: "{{ view_path }}"
+                target:
+                  device_id: "{{ target_display_device }}"
+                action: browser_mod.navigate
+              - alias: >-
+                  Set enqueue_type, media_type and response_first_word depending
+                  on cuetrack or track trigger
+                choose:
+                  - conditions:
+                      - condition: trigger
+                        id:
+                          - cuetrack
+                    sequence:
+                      - variables:
+                          enqueue_type: next
+                          media_type: track
+                          response_first_word: Queueing
+                  - conditions:
+                      - condition: trigger
+                        id:
+                          - track
+                    sequence:
+                      - variables:
+                          enqueue_type: play
+                          media_type: track
+                          response_first_word: Playing
+              - target:
+                  entity_id: "{{ target_musicplayer_device }}"
+                data:
+                  media_type: "{{media_type}}"
+                  enqueue: "{{enqueue_type}}"
+                  media_id: "{{ found_song.tracks.0.name }}"
+                  artist: "{{ trigger.slots.artist  }}"
+                action: music_assistant.play_media
+                enabled: true
+              - set_conversation_response: >-
+                  {{response_first_word}} the song {{found_song.tracks.0.name}}
+                  by {{trigger.slots.artist}} now!
+            else:
+              - set_conversation_response: >-
+                  I was unable to find any song called {{trigger.slots.song}} by
+                  {{ trigger.slots.artist }}
 variables:
-  view_path: !input view_path
-  group_entity: !input group_entity
-  ma_instance: !input ma_instance
+  view_path: /dashboard-viewassist/music
```

Comment:
please revert this and the next two as they are inadvertently set to your specifics.